cedar-policy-validator/src/type_error.rs

```diff
@@ -289,6 +289,7 @@ pub enum TypeErrorKind {
     #[error("empty set literals are forbidden in policies")]
     EmptySetForbidden,
     #[error("extension constructors may not be called with non-literal expressions")]
+    #[diagnostic(help("consider applying extension constructors to attributes when constructing entity or context data"))]
```

Comment:
Maybe this? In the current wording, I'm not sure what "attributes" refers to.
```suggestion
    #[diagnostic(help("consider applying extension constructors to literal values when constructing entity or context data"))]
```

Reply:
(Also, yay for one line PRs! So easy to review 😄)

Reply:
I think the point here was not to emphasize "use literal values" (that was clear from the error message itself), but to emphasize "do it when constructing the entity or context data [and not in the policy]".  This could probably still be clearer